### PR TITLE
v2.2.1 Release

### PR DESCRIPTION
**Pull request by rstoneback**
# Description

Addresses # (issue)

Includes bug fixes for documentation on ReadTheDocs.

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Documentation viewed on ReadTheDocs website.

# Checklist:

- [ ] Make sure you are merging into the ``develop`` (not ``master``) branch
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

File: docs/api.rst [hunk 1]
```diff
@@ -8,6 +8,83 @@ Instrument
...
+   :members: __doc__, download, bytes_to_float, load_general_header, load_location_parameters, load_attitude_parameters, load_binary_file, set_metadata
+
+
+.. _rst_general_data_general:
+
+General
+^^^^^^^
+
+.. automodule:: pysat.instruments.methods.general
+   :members:
```
Comment (jklenzing) [inline docs/api.rst:30]: no members in general?

File: docs/api.rst [hunk 2]
```diff
@@ -8,6 +8,83 @@ Instrument
...
+.. automodule:: pysat.instruments.methods.nasa_cdaweb
+   :members: __doc__, load, list_files, list_remote_files, download
+
+.. _rst_general_data_madrigal:
+
+NASA ICON
+^^^^^^^^^
+
+.. automodule:: pysat.instruments.methods.icon
+   :members:
```
Comment (jklenzing) [inline docs/api.rst:46]: ?

File: docs/api.rst [hunk 3]
```diff
@@ -8,6 +8,83 @@ Instrument
...
+^^^^^^^^
+
+.. automodule:: pysat.instruments.methods.madrigal
+   :members: __doc__, cedar_rules, load, download, filter_data_single_date
+
+Space Weather
+^^^^^^^^^^^^^
+
+.. automodule:: pysat.instruments.methods.sw
+   :members:
```
Comment (jklenzing) [inline docs/api.rst:58]: ?

File: docs/api.rst (hunk 1 above)
Comment (rstoneback) [reply docs/api.rst:30]: Left those blank so the docs would pull in everything it found.

**Pull request merged**